packages/api-server/vitest.config.mts

```diff
@@ -0,0 +1,21 @@
+import { defineConfig, configDefaults } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    exclude: [
+      ...configDefaults.exclude,
+      '**/fixtures',
+      // Dist tests are treated differently and not run in the normal test suite
+      './dist.test.ts'
+    ],
+  },
+  plugins: [
+    {
+      name: 'virtual-modules',
+      resolveId(id) {
+        if (id === '/redwood-app/api/server.config.js' || id === '\\redwood-app\\api\\server.config.js')
```

Comment:
Feels like this should have a line break. Is prettier not formatting this file?

Reply:
Nope, nothing happens automatically or when I request it to be formatted.